mlrun/projects/pipelines.py

```diff
@@ -593,7 +593,7 @@ def run(
             logger.warning(
                 "Setting notifications on kfp pipeline runner uses old notification behavior. "
                 "Notifications will only be sent if you wait for pipeline completion. "
-                "To use the new notification behavior, use the remote pipeline runner."
+                "Some of the features (like setting message or severity level)."
```

Comment:
Some of the features what? 😄 
Guess you mean "Some of the features are not supported" ?